core-bundle/src/Security/User/UserChecker.php

```diff
@@ -50,9 +50,6 @@ public function checkPostAuth(UserInterface $user): void
     {
     }
 
-    /**
-     * @throws LockedException
-     */
```

Comment:
This should stay, I guess. The function has bascially a `never` return type for a reason.

Following the logic of my last comment, the exception would need to be added to `checkPreAuth()` as well. I do know too less of the context of this code, but I assume a consumer calling the public method will need to handle `LockedExceptions`?

Reply:
(same for some other private methods above, but I did not want to spam the changed files view)

Reply:
No, it should be removed.

Keeping the annotation would imply "please handle this exception by wrapping the method call in a try/catch block". That is exactly what we don't want. We want the exception to bubble up.